notebooks/parsing_mbox.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<p>With notebooks, it is often good practice to create a Markdown cell where you include the "notes" regarding what you are doing in code, instead of including the #comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>